### DISCUSSION TITLE
My changes seem to be merged, but they aren't

### DISCUSSION THREAD

**Discussion by gorkem-bwl** [GitbookIO/community#821] [3. Git Sync]
Hi guys,

Five days ago, I updated my documentation and confirmed the changes were reflected correctly on the documentation URL. However, when I checked today, the changes are missing—it looks like they’ve been reverted.

For what it's worth, I can see that the November 29 changes are marked as "merged," but when I check GitHub, those changes aren’t there. Is there a way to view the "November 29" changes as they appear on the dashboard, even though they’re not showing up on the public documentation page?

I’m wondering if I might have misunderstood something or made a mistake.

Thanks for the help!

**Comment by aikoii** [comment]
Hey @gorkem-bwl, could you share the space ID/site URL so we can investigate further?

**Reply by gorkem-bwl** [reply]
> Hey @gorkem-bwl, could you share the space ID/site URL so we can investigate further?

Site ID is "probably" site_47uuo and the URL is https://bluewavelabs.gitbook.io/checkmate

**Reply by aikoii** [reply]
Thanks! I'm looking into it

**Reply by gorkem-bwl** [reply]
Thanks a ton! A lot of changes went into that 29Nov branch, but I have no idea how to make them visible :)

**Reply by gorkem-bwl** [reply]
Have you been able to find anything interesting @aikoii ? :)

**Reply by aikoii** [reply]
Hey @gorkem-bwl, appreciate your patience! 🙏

From what the team has shared, this isn't related to content loss after merging a change request, but rather due to the import from the GitHub repo. If you check the Git sync history, you'll see that the last import on this space led to the removal of the Server Monitoring Agent page. If you take a look at the [GitHub repo](https://github.com/bluewave-labs/checkmate/tree/develop/guides/users-guide), you'll notice that the page didn’t actually exist there. I hope this clears things up!

**Reply by gorkem-bwl** [reply]
Thanks for looking at this. I see it on the page, so I believe if I can copy the previous contents I should be good to go here.

**Reply by gorkem-bwl** [reply]
One more thing - I am trying to edit the page now, and this shows up (seeing it for the last 3-4 minutes or so)

<img width="406" alt="image" src="https://github.com/user-attachments/assets/27ba086b-06ec-4cf8-8c3a-c1efe6cc35b6" />

For how long do you think I have to wait before merging those changes?

**Reply by aikoii** [reply]
Hmm, this is odd - it shouldn't really take that long. It can take a bit longer depending on the size of the change, but not 3-4 minutes.

Next time it happens (I hope it doesn't! 🤞), could you share a recording with your network console open? 🙏
